authors/apps/authentication/views.py

```diff
@@ -188,11 +188,11 @@ def get(self, request, **kwargs):
         data = jwt.decode(hashed_email, SECRET_KEY, algorithm='HS256')
         user = User.objects.get(email=data['email'])
         if not default_token_generator.check_token(user, token):
-            return Response({"response": 'Invalid reset tokens'},
-                            status=status.HTTP_401_UNAUTHORIZED)
+            return Response({"response": 'Invalid token'},
+                            status=status.HTTP_400_BAD_REQUEST)
```

Comment:
I think the status code here should be `401` because if authentication fails, that means the user is unauthorized.
cc: @wrotich @abadojack @evamaina @TheSteelGuy @Eyansky